docs/user/quickstart.rst

```diff
@@ -69,8 +69,7 @@ templates, along with the CSS and JS files necessary to make it work:
 
         <script src="https://code.jquery.com/jquery-1.9.1.min.js"></script>
         <script src="https://login.persona.org/include.js"></script>
```

Comment:
`browserid_js` should include the shim by default too, so this isn't necessary either.